CHANGELOG.rst

```diff
@@ -67,6 +68,7 @@ Fixed
 - ``rasa interactive`` now looks for endpoints in ``endpoints.yml`` if no ``--endpoints`` arg is passed
 - custom files, e.g. custom components and channels, load correctly when using
   the command line interface
+- ``MappingPolicy`` now works correctly when used as part of a PolicyEnsemble
```

Comment:
Where did this happen?

Reply:
it was supposed to not be part of 1.1.5 because the first tag was before it was merged, but it got merged while we were fighting travis. so technically the change actually made it into 1.1.5